export/caches/object_cache.py

```diff
@@ -86,7 +86,10 @@ def _debug_info(self):
     def _is_visible(self, dg_obj_instance, obj):
         # TODO if this code needs to be used elsewhere (e.g. in material preview),
         #  move it to utils (it doesn't concern this cache class)
-        return dg_obj_instance.show_self and obj.type in EXPORTABLE_OBJECTS
+        if obj.luxcore.exclude_from_render:
+            return not obj.luxcore.exclude_from_render and obj.type in EXPORTABLE_OBJECTS
+        else:
+            return dg_obj_instance.show_self and obj.type in EXPORTABLE_OBJECTS

```

Comment:
This looks overly complicated to me, why not just:
```python
return dg_obj_instance.show_self and not obj.luxcore.exclude_from_render and obj.type in EXPORTABLE_OBJECTS
```

Reply:
> > This one isn't refreshed properly in the viewport (it updates fine to set the bool to false, but not to set it to true).
> 
> Have you tried what I suggested here?
> [#331 (comment)](https://github.com/LuxCoreRender/BlendLuxCore/issues/331#issuecomment-564771998)

I tried but that didn't work unfortunately.

Reply:
> This looks overly complicated to me, why not just:
> 
> ```python
> return dg_obj_instance.show_self and not obj.luxcore.exclude_from_render and obj.type in EXPORTABLE_OBJECTS
> ```

No reason except that I'm a noob. I tried "or" but that wouldn't work, didn't even think of using "and". Thanks for reviewing.